static/docs/user-guide/dvc-file-format.md

```diff
@@ -60,3 +64,10 @@ A metric entry consists of such fields:
 - `type`: type of the metrics file (e.g. raw/json/tsv/htsv/csv/hcsv);
 - `xpath`: path within the metrics file to the metrics data(e.g. `AUC.value` for
   `{"AUC": {"value": 0.624321}}`);
+
+A meta entry consists of `key:value` pairs such as `name: john`. A meta entry
+can have any structure and contain any number of attributes. `"meta: string"` is
+also possible, it doesn't necessarily need to contain dictionary always.
+
+Comments can be added to the .dvc file using `# comment` syntax. Comments are
+preserved between multiple execution of  `dvc repro <filename.dvc>` command.
```

Comment:
of the `dvc repro` command, no need for <filename.dvc> - it's optional.

probably we need to use `multiple executions`

need to clarify, if meta and comments are preserved when we do `dvc run` that overwrites the file again - please, clarify it on #dev-general

Reply:
the same for `dvc add` and `dvc import` - they also create DVC-files. We need to clarify here, on in their documentation what happens to comments/meta if we run them again.

also, `dvc commit` updates DVC-files - let's clarify if it also preserves the comments.

Reply:
> need to clarify, if meta and comments are preserved when we do `dvc run` that overwrites the file again - please, clarify it on #dev-general

meta and comments are not preserved between `dvc run` command.